packages/mobile/src/home/CashInBottomSheet.tsx

```diff
@@ -54,15 +124,35 @@ function CashInBottomSheet() {
         >
           <Times />
         </Touchable>
-        <Text style={styles.title}>{t('cashInBottomSheet.title')}</Text>
-        <Text style={styles.subtitle}>{t('cashInBottomSheet.subtitle')}</Text>
-        <Button
-          text={t('cashInBottomSheet.addFunds')}
-          size={BtnSizes.FULL}
-          onPress={goToAddFunds}
-          style={styles.addFundBtn}
-          testID={'cashInBtn'}
-        />
+        {rampAvailable && rampCashInButtonExpEnabled ? (
+          <>
+            <Text style={styles.title}>
+              {t('cashInBottomSheet.titleRamp', {
+                currency: localCurrency === LocalCurrencyCode.EUR ? Currency.Euro : Currency.Dollar,
```

Comment:
it annoys me that Currency and CiCoCurrency are different.. but that's not your fault

Reply:
ikr!